setup.py

```diff
@@ -52,9 +42,18 @@
     anchore-cli=anchorecli.cli:main_entry
     ''',
     install_requires=requirements,
-    scripts=scripts
+    scripts=scripts,
+    classifiers=[
```

Comment:
if this project is supporting multiple Python versions it is important to set the classifiers so that they show up on PyPI